src/DebugSceneList.js

```diff
@@ -120,12 +122,18 @@ class DebugSceneList extends Component {
   }
 
   renderSceneModal() {
+    let component = this.state.selectedItem.component;
+    const closePropName = this.state.selectedItem.closePropName;
+    console.log('closePropName', closePropName);
+    if (closePropName) {
+      component = React.cloneElement(component, {[closePropName]: this.onHideScene});
+    }
     return (
```

Comment:
While I was out getting lunch I had a *much* better idea, way easier to document too:
* if you pass in a rendered component, we show it as-is
* if you pass in a component, we render it with a fixed set of props

so then instead of doing this weird `closePropName` thing, there is a much more civilised interface that looks like:

```js
addSceneTest(({onClose}) => <MyModal onRequestClose={onClose} />, {
  name: 'MyModal',
});
```

which just has the one caveat that, if you're passing in an anonymous component like that in order to get the onClose parameter, then you pretty much always need to supply an explicit 'name' parameter.